pages/web/prax.mdx

```diff
@@ -1,3 +1,5 @@
+import WalletInstall from '@/components/WalletInstall';
+
 # Prax Wallet
```

Comment:
suggestion: on the top-level [web](https://guide.penumbra.zone/web) page, it could be helpful to mention that this is an interactive, step-by-step guide intended to be followed in order.

This relates to my Slack comment about the loading indicators on the "Shield Funds," "Swapping Tokens," "Staking," and "Disconnect" pages showing indefinite loading. I intentionally skipped steps to see what would happen, and adding a **note** about this might help prevent confusion.

Reply:
maybe something like

`"Welcome to the interactive guide for the Prax Wallet web extension. This step-by-step tutorial is designed to help you navigate and use the Prax Wallet effectively. Please follow each section in order to ensure the best experience."`

Reply:
Agree, adding a note to the top-level page.